Makefile

```diff
@@ -1,19 +1,34 @@
 CC = cl
 LD = link
+RC = rc 
 
 EXEC = lightwm.exe
 DLL = lightwm_dll.dll
+RESOURCES=wm_dll_resources.obj
 
-all: clean $(DLL) $(EXEC)
+# Boiler plate
+CFLAGS  = /EHsc /Zi
```

Comment:
By default, it should build as Release without the debugging flag, for debugging - I would recommend adding another rule to the make file that will build as Debug.